clients/go/commands/failJob_command.go

```diff
@@ -15,7 +15,12 @@ type FailJobCommandStep1 interface {
 }
 
 type FailJobCommandStep2 interface {
-	Retries(int32) DispatchFailJobCommand
+	Retries(int32) FailJobCommandStep3
+}
+
+type FailJobCommandStep3 interface {
+	DispatchFailJobCommand
+	ErrorMessage(string) FailJobCommandStep3
```

Comment:
I assume returning `DispatchFailJobCommand` makes more sense?

Reply:
I decided to return `FailJobCommandStep3` to allow us to add more optional methods in case we need it. Similar to the Java API.

Reply:
But this is still possible, because Retries returns `FailJobCommandStep3`. The thing is now it is possible to call `ErrorMessage` twice which makes no sense, since the last call will overwrite the first one.

Reply:
Ok I think I get it. You mean that if you call ErrorMessage you can only call send not new optional parameters which are added. Ja ok lets stay with this solution

Reply:
> The thing is now it is possible to call ErrorMessage twice which makes no sense, since the last call will overwrite the first one.

That is also true for the Java API for all the last steps in the builder pattern. Its not really possible to have optional parameters without allowing them to be called twice or?

Reply:
I think only with new interfaces.